CONTRIBUTING.md

```diff
@@ -101,3 +101,16 @@ Please note that the recommended local port for the services are different than
     - All dependencies must be abstracted as an interface. This will make it easier to switch libraries at later point
       of time.
 * Version tagging as per Semantic versioning (https://semver.org/)
+
+### Documentation
+* After adding or modifying code existing code, update the documentation too - [development/docs](https://github.com/gofr-dev/gofr/tree/development/docs).
+* When you consider a new documentation page is needed, start by adding a new file and writing you new documentation. Then - add a reference to it in [navigation.js](https://gofr.dev/docs/navigation.js).
```

Comment:
`writing you new do` grammatically it is incorrect

Reply:
you -> your